src/blocks/weather.rs

```diff
@@ -38,6 +38,17 @@
 //! in which case they must be provided in the environment variables
 //! `OPENWEATHERMAP_API_KEY`, `OPENWEATHERMAP_CITY_ID`, `OPENWEATHERMAP_PLACE`.
 //!
+//! # met.no Options
+//!
+//! Key | Values | Required | Default
+//! ----|--------|----------|--------
+//! `name` | `metno`. | Yes | None
+//! `coordinates` | GPS latitude longitude coordinates as a tuple, example: `["39.236229089090216","9.331730718685696"]`
+//! `lang` | Language code: `en`, `nn` or `nb` | No | `en`
+//! `altitude` | Meters above sea level of the ground | No | Approximated by server
+//!
+//! Met.no does not support location name or apparent temperature.
```

Comment:
I suppose we can re-add custom apparent temperature calculation function which was removed in #1573

Reply:
Okay, done in 631e81a.